repo/packages/Q/quobyte/0/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "quobyte",
+  "version": "1.2",
+  "tags": [
+    "storage",
+    "filesystem",
+    "bigdata"
+  ],
+  "maintainer": "silvan@quobyte.com",
+  "description": "Quobyte is a software defined distributed file system, running on commodity hardware.",
+  "framework": true,
+  "scm": "https://github.com/bar/foo.git",
+  "website": "http://www.quobyte.com/",
+  "images": {
+    "icon-small": "http://www.quobyte.com/mesos/icon-quobyte-small.png",
+    "icon-medium": "http://www.quobyte.com/mesos/icon-quobyte-medium.png",
+    "icon-large": "http://www.quobyte.com/mesos/icon-quobyte-large.png",
+    "screenshots": [
+      "http://www.quobyte.com/mesos/screen-1.png",
+      "http://www.quobyte.com/mesos/screen-2.png",
+      "http://www.quobyte.com/mesos/screen-3.png"
+    ]
+  },
+  "preInstallNotes": "Installing Quobyte.\nPlease ensure your devices have been set up (see https://github.com/quobyte/mesos-framework#setup-devices)",
```

Comment:
looking for the preInstall notes to inform user of resource requirements... 1 core and 512M etc.